### DISCUSSION TITLE
How to overwrite nerd font icons?

### DISCUSSION THREAD

**Discussion by MoltenMonster** [ryanoasis/nerd-fonts#1723] [Q&A]
For instance, I want to replace the `nf-md-numeric_1_circle_outline` icon (you can search that in [https://www.nerdfonts.com/cheat-sheet](https://www.nerdfonts.com/cheat-sheet) to see), with this Apple [SF Symbol](https://developer.apple.com/sf-symbols/) `1.circle` instead. I use a nerd font in Neovide (an Nvim GUI), where I can paste in the text character for `1.circle` and it shows correctly without additional steps. On [nerdfonts.com](https://nerdfonts.com), it says at the bottom "You can even specify a custom symbol font with the `--custom` option to include even more glyphs." but I didn't find any instructions for how exactly to make a font file with the glyphs I want. I want to be able to overwrite any nerd icon with one in the SF Symbols app.

![CleanShot 2024-10-28 at 16 50 28@2x](https://github.com/user-attachments/assets/025556e8-fd73-4b57-8f02-7f273a11a152)

**Comment by Finii** [comment]
* Take the SF Symbol "1.circle" and put it into a new font
* at the codepoint of `numeric_1_circle_outline`
* Patch a font and add `--custom your_new_font_file.ttf` to all the other options you use

All glyphs in the `your_new_font_file.ttf` font will be patched in without shifting codepoints and without scaling.

If you already have a Nerd Fonts patched font it might be easier to do that with a font editor like `fontforge`
* Open both fonts
* Mark 1.circle in SF Symbol font and click `copy`
* Mark destination codepoint in target font
* Push `paste`
* Save font

:grimacing: 

> I use a nerd font in Neovide (an Nvim GUI), where I can paste in the text character for 1.circle and it shows correctly without additional steps.

Not sure what that means :thinking: If it works, it works?
